toolkit-test-app/src/main/java/com/esri/arcgisruntime/toolkit/test/sceneview/ArcGISSceneViewTableTopActivity.kt

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.test.sceneview
+
+import android.graphics.Point
+import android.os.Bundle
+import android.support.annotation.StringRes
+import android.support.design.widget.Snackbar
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.view.MotionEvent
+import android.widget.Toast
+import com.esri.arcgisruntime.layers.Layer
+import com.esri.arcgisruntime.layers.PointCloudLayer
+import com.esri.arcgisruntime.loadable.LoadStatus
+import com.esri.arcgisruntime.mapping.ArcGISScene
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource
+import com.esri.arcgisruntime.mapping.NavigationConstraint
+import com.esri.arcgisruntime.mapping.Surface
+import com.esri.arcgisruntime.mapping.view.Camera
+import com.esri.arcgisruntime.mapping.view.DefaultSceneViewOnTouchListener
+import com.esri.arcgisruntime.toolkit.ar.ArcGISArView
+import com.esri.arcgisruntime.toolkit.extension.logTag
+import com.esri.arcgisruntime.toolkit.test.R
+import kotlinx.android.synthetic.main.activity_arcgissceneview.arcGisArView
+import java.net.URI
+
+class ArcGISSceneViewTableTopActivity : AppCompatActivity(), ArcGISArView.OnStateChangedListener {
+
+    private var _scene: ArcGISScene? = null
+    private var pointCloudLayer: Layer? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_arcgissceneview_tabletop)
+        arcGisArView.registerLifecycle(lifecycle)
+        arcGisArView.addOnStateChangedListener(this)
+
+        arcGisArView.sceneView.setOnTouchListener(object : DefaultSceneViewOnTouchListener(arcGisArView.sceneView) {
+            override fun onSingleTapConfirmed(motionEvent: MotionEvent?): Boolean {
+                motionEvent?.let {
+                    if (_scene != null && pointCloudLayer != null && !_scene!!.operationalLayers.contains(
+                            pointCloudLayer
+                        )
+                    ) {
```

Comment:
Can this all go on one line?

Reply:
It could but it goes over the 100 columns limit so everytime I run the code cleanup it'll reset it. It might look weird but it's formatted to Kotlin styles.